### PR TITLE
Add support for Open edX Olive release

### PR DESCRIPTION
**Pull request by mrtmm**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def load_readme():
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
     python_requires=">=3.6",
-    install_requires=["tutor <15, >=14.0.0"],
+    install_requires=["tutor <16, >=14.0.0"],
```
Comment (fghaas) [inline setup.py:28]: Shouldn't this be:

```suggestion
    install_requires=["tutor <16, >=15.0.0"],
```

**Review by fghaas — COMMENTED** [review]
I notice that in contrast to the PRs on the other plugins which are drafts, this one was immediately marked ready for review, so review I did. :)

File: setup.py (hunk 1 above)
Comment (mrtmm) [reply setup.py]: Yes it should :)

**Review by fghaas — APPROVED** [review]
This looks good to me now. Should we go ahead and merge this to `main`, and create the `nutmeg` branch from the prior commit?

**Comment by mrtmm** [comment]
Yes, I _think_ for this one we should be good.

**Comment by fghaas** [comment]
Yeah wait, I need to check on the GitHub Actions workflow first. (The commit message in this commit _should_ fail `gitlint`, but it doesn't. I'll look into that.)

**Comment by fghaas** [comment]
@mrtmm This is now rebased on current `main`, and by having [0987ae6](https://github.com/hastexo/tutor-contrib-enrollmentreports/pull/18/commits/0987ae6c0988f714c7b195b20492a939cf9cf43f) as a separate commit we can split the `nutmeg` branch at _that_ commit and the README will be correct for both that branch and `main`. Does that all sound good to you?

**Comment by mrtmm** [comment]
Yes, sounds good to me, thanks!

**Review by fghaas — APPROVED** [review]

**Comment by fghaas** [comment]
All done. Merged this one to `main`, and the `nutmeg` branch currently points to https://github.com/hastexo/tutor-contrib-enrollmentreports/pull/18/commits/0987ae6c0988f714c7b195b20492a939cf9cf43f.

**Pull request merged**